plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/ui/CompositeUtil.java

```diff
@@ -33,6 +34,15 @@ public boolean apply(Control control) {
     });
   }
 
+  public static Button findButton(Composite composite, final String text) {
```

Comment:
TBH we should minimize these types of methods and use SWTBot's helpers (e.g., `SWTBot#buttonWithLabel()`).

Reply:
Having trouble making `buttonWithLabel()`. Not sure why it doesn't work.